### PR TITLE
Rasmus Persson & Julia Ros 

### PR DESCRIPTION
**Pull request by rasmuselmgren**
This is a colab project with myself and Julia Ros

Link to my deployed project: https://top-moviess.netlify.app/

### CODE REVIEW

File: code/src/pages/MovieList.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  useEffect(() => {
+    fetch(API_URL)
+      .then((res) => res.json())
+      .then((json) => {
+        setMovies(json.results);
+      });
+  }, []);
+
+  return (
+    <Grid container justifiy="center">
```
Comment (Aepera) [inline code/src/pages/MovieList.js:21]: Is "justifiy" misspelled?

File: code/src/pages/MovieDetails.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    fetch(API_DETAILS_URL)
+      .then((response) => response.json())
+      .then((json) => setMovieDetails(json));
+  }, [API_DETAILS_URL]);
+
+  return (
+    <div className="position-relative">
+      <Link to="/">
+        <div className="back-link">
+          <i className="fas fa-chevron-circle-left"></i>&nbsp;
```
Comment (Aepera) [inline code/src/pages/MovieDetails.js:22]: Good solution for the back-button with font awesome.

File: code/src/components/MovieThumb.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+            direction="column"
+            justify="flex-end"
+            className="position-absolute hidden-shown text-realeased-content"
+          >
+            <h1>{title}</h1>
+            <h5>Released {release_date}</h5>
+          </Grid>
+          <img
+            className="movie-thumb-img"
+            src={`https://image.tmdb.org/t/p/w780${poster_path}`}
```
Comment (Aepera) [inline code/src/components/MovieThumb.js:21]: Nice and clean solution to create the url here! 🎉

File: code/src/index.css [hunk 4]
```diff
@@ -5,9 +5,145 @@ body {
...
+  font-weight: bold;
+  font-size: medium;
+  display: flex;
+  align-items: center;
+  text-decoration: none;
+}
+
+.back-link-margin:hover {
+  margin-left: 5px;
+  transition: 0.2s ease-in-out;
```
Comment (Aepera) [inline code/src/index.css:71]: 👌

**Review by Aepera — APPROVED** [review]
Great job with this project! The website is working as it should and we cannot find anything not working. 
The code is easy to read and looks clean. Well done! 😃

**Review by KarolCislowski — APPROVED** [review]
Great project 🦄

**Pull request closed without merging**